packages/amplify_datastore/android/src/main/kotlin/com/amazonaws/amplify/amplify_datastore/types/hub/FlutterSyncMetaData.kt

```diff
@@ -27,7 +27,7 @@ class FlutterSyncMetaData(var metadata: OutboxMutationEvent.OutboxMutationEventE
                 "id" to this.metadata.model.id,
                 "_deleted" to this.metadata.isDeleted,
                 "_version" to this.metadata.version,
-                "_lastChangedAt" to this.metadata.lastChangedAt
+                "_lastChangedAt" to (this.metadata.lastChangedAt ?: null)
```

Comment:
is this needed?

Reply:
Yes - it specifically resolved an issue (I can't recall off the top of my head what it was).